docs/source/merlin_commands.rst

```diff
@@ -29,8 +29,8 @@ The Merlin can be output using the ``-v`` argument.
     (merlin3_7) $ merlin -v
 
 
-Information
------------
+Information (info)
```

Comment:
i like these additions
let's be more explicit "Information (`merlin info`)" etc

Reply:
Will do